code/modules/lighting/emissive_blocker.dm

```diff
@@ -9,7 +9,7 @@
 /atom/movable/emissive_blocker
 	name = ""
 	plane = EMISSIVE_BLOCKER_PLANE
-	layer = EMISSIVE_BLOCKER_LAYER
+	layer = 0
```

Comment:
Is this line even needed anymore?

Reply:
no, this was the result of a regex to find and replace these in add vis overlays.